Demo/Fullscreen/FrontPage/FrontPageViewDemoView.swift

```diff
@@ -44,7 +45,17 @@ extension FrontpageViewDemoView: FrontPageViewDelegate {
 }
 
 extension FrontpageViewDemoView: AdsGridViewDelegate {
-    public func adsGridView(_ adsGridView: AdsGridView, willDisplayItemAtIndex index: Int) {}
+    public func adsGridView(_ adsGridView: AdsGridView, willDisplayItemAtIndex index: Int) {
+        if index >= visibleItems - 10 {
+            visibleItems += 10
+
+            DispatchQueue.main.asyncAfter(deadline: .now() + 0.5, execute: { [weak self] in
+                self!.frontPageView.reloadAds()
```

Comment:
Why not `self?`?

Reply:
Ah snap, I was testing something out and forgot to clean up after myself. Will fix!!!